pkg/install/config.go

```diff
@@ -8,14 +8,58 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type arkConfigOption func(*arkConfig)
+
+type arkConfig struct {
+	backupSyncPeriod   time.Duration
+	gcSyncPeriod       time.Duration
+	scheduleSyncPeriod time.Duration
+	restoreOnly        bool
+}
+
+func WithBackupSyncPeriod(t time.Duration) arkConfigOption {
+	return func(c *arkConfig) {
+		c.backupSyncPeriod = t
+	}
+}
+
+func WithGCSyncPeriod(t time.Duration) arkConfigOption {
+	return func(c *arkConfig) {
+		c.gcSyncPeriod = t
+	}
+}
+
+func WithScheduleSyncPeriod(t time.Duration) arkConfigOption {
```

Comment:
We're planning on removing this configuration option and instead hardcoding it to 1 minute, so we may want to omit this.

Reply:
OK, i'll remove the option for now and leave the resource specification hardcoded to 1 minute.